src/main.rs

```diff
@@ -118,6 +132,27 @@ struct Args {
     cmd_config: bool,
 }
 
+impl Args {
+    fn action(&self) -> Action {
+        // extract operation mode
+        let action = if self.cmd_fix || self.flag_fix {
+            Action::Fix
+        } else if self.cmd_reflow {
+            Action::Reflow
+        } else if self.cmd_config {
+            Action::Config
+        } else if self.flag_help {
```

Comment:
It seems to me that this could lead to `--help` not being recognised fi `fix` is present.

Reply:
Nevermind, docopt does not allow both flags

Reply:
Exactly :) and this is necessary for the case where `cargo-spellcheck` is invoked via `cargo spellcheck` without further arguments.